.gitignore

```diff
@@ -12,3 +12,5 @@ develop-eggs
 .installed.cfg
 lib
 lib64
+
```

Comment:
The files before it are generated while building packages. 
.idea isn't. Thus, separated it